includes/reader-activation/class-reader-activation.php

```diff
@@ -130,6 +130,7 @@ public static function enqueue_scripts() {
 				'otp_auth_action'       => Magic_Link::OTP_AUTH_ACTION,
 				'otp_rate_interval'     => Magic_Link::RATE_INTERVAL,
 				'account_url'           => function_exists( 'wc_get_account_endpoint_url' ) ? \wc_get_account_endpoint_url( 'dashboard' ) : '',
+				'ras_is_enabled'        => self::is_enabled(),
```

Comment:
non-blocking nit: Would `is_ras_enabled` be a better name here? It follows the naming convention for other boolean flags.